### ISSUE TITLE
VSCode 1.41 - Custom CSS not being applied 

### ISSUE THREAD

**Issue by heiy** [be5invis/vscode-custom-css#109]
macOS 10.14 
vsCode 1.41

my config path are right

```
    "vscode_custom_css.imports": [
        "file:///Users/userName/Documents/custom.css",
        "file:///Users/userName/Documents/custom.js"
    ],
 "vscode_custom_css.policy": true,
```

If I "Reload Custom CSS and JS", I get the notification to restart VSCode. I restart and nothing changes. I still get the "Your Code installation appears to be corrupt" error though :(

If I check the devtools console, I see:
```
ERR 无法读取文件(Error: 找不到文件(gitfs:/Users/zhangcenfeng/Documents/custom.css.git?path=%2FUsers%2Fzhangcenfeng%2FDocuments%2Fcustom.css&ref=)): Error: 无法读取文件(Error: 找不到文件(gitfs:/Users/zhangcenfeng/Documents/custom.css.git?path=%2FUsers%2Fzhangcenfeng%2FDocuments%2Fcustom.css&ref=))
    at _.doReadAsFileStream (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:1591:434)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:89:5)
```
my css

```
html {
  background: transparent !important;
}
 
.scroll-decoration {
  box-shadow: none !important;
}
 
.minimap {
  opacity: 0.6;
}
 
.editor-container {
  background: transparent !important;
}
 
.search-view .search-widget .input-box, .search-view .search-widget .input-box .monaco-inputbox,
.monaco-workbench>.part.editor>.content>.one-editor-silo>.container>.title .tabs-container>.tab,
.monaco-editor-background,
.monaco-editor .margin,
.monaco-workbench>.part>.content,
.monaco-workbench>.editor>.content>.one-editor-silo.editor-one,
.monaco-workbench>.part.editor>.content>.one-editor-silo>.container>.title,
.monaco-workbench>.part>.title,
.monaco-workbench,
.monaco-workbench>.part,
body {
  background: transparent !important;
}
 
.editor-group-container>.tabs {
  background-color: rgba(37, 37, 37,0.2) !important;
}
 
.editor-group-container>.tabs .tab {
  background-color: transparent !important;
}
 
.editor-group-container>.tabs .tab.active {
  background-color: rgba(37, 37, 37,0.4) !important;
}
 
.monaco-list.settings-toc-tree .monaco-list-row.focused {
  outline-color: rgb(37, 37, 37,0.6) !important;
}
 
.monaco-list.settings-toc-tree .monaco-list-row.selected,
.monaco-list.settings-toc-tree .monaco-list-row.focused,
.monaco-list .monaco-list-row.selected,
.monaco-list.settings-toc-tree:not(.drop-target) .monaco-list-row:hover:not(.selected):not(.focused) {
  background-color: rgb(37, 37, 37,0.6) !important;
}
 
.monaco-list.settings-editor-tree .monaco-list-row {
  background-color: transparent !important;
  outline-color: transparent !important;
}
 
.monaco-inputbox {
  background-color: rgba(41, 41, 41,0.2) !important;
}
 
.monaco-editor .selected-text {
  background-color: rgba(160, 171, 187, 0.7) !important;
}
 
.monaco-editor .focused .selected-text {
  background-color: rgba(172, 172, 172, 0.6) !important;
}
 
.mtk4{ color:#6c77b7}
 
.monaco-editor .view-overlays .current-line {
  border-color: rgba(41, 41, 41,0.2) !important;
}
 
.extension-editor,
.monaco-inputbox>.wrapper>.input,
.monaco-workbench>.part.editor>.content>.one-editor-silo>.container>.title .tabs-container>.tab.active,
.preferences-editor>.preferences-header,
.preferences-editor>.preferences-editors-container.side-by-side-preferences-editor .preferences-header-container,
.monaco-editor, .monaco-editor .inputarea.ime-input {
  background: transparent !important;
}
 
.editor-group-container>.tabs .tab {
  border: none !important;
}
```

**Comment by heiy** [comment]
I find custom.js  not being applied
```
nodeRequire('electron').remote.getCurrentWindow().setVibrancy('ultra-dark');
```

**Comment by MattJett-EWU** [comment]
I'm having the same problem

**Comment by icepole-jon** [comment]
Windows File URL Example: file:///C:/Users/MyUserName/Documents/custom.css

The C:/ part is REQUIRED.

你少了盘符

**Comment by heiy** [comment]
VSCode said 1.36+  not support 
`nodeRequire('electron').remote.getCurrentWindow().setVibrancy('ultra-dark');`

**Comment by chobijaeyu** [comment]
same problem

**Comment by justinchristensen66** [comment]
I was having trouble on my MacOS as well but after entering the custom.css.imports location ""file:///Users/userName/Documents/custom.css"" **,1** I opened the terminal and and entered.      sudo chown -R username /usr/local/bin/code,  the last part of the command is your path to VSCode on your machine and the second part is your username on the computer. **2** Then I closed VSCode and reopened it. **3** Then I Disabled(skipped the prompt to restart) and then Enabled Custom CSS and JS, was prompted to restart VSCode, I hit restart and it worked.  I watched someone on youtube do it, although the video was not in english, I could still follow along.  [https://youtu.be/7ZvGqzXl_Nk](url) The glowing effect is missing on some words but appears on some as well, it doesn't look exactly like the picture on theme plugin but it still seems to work.
